vernac/vernacentries.ml

```diff
@@ -2250,32 +2258,36 @@ let with_fail st b f =
       | _ -> assert false
   end
 
+let attributes_of_flags f atts =
+  List.fold_left
+    (fun (polymorphism, atts) f ->
+       match f with
+       | VernacProgram when not atts.program ->
+         (polymorphism, { atts with program = true })
+       | VernacProgram ->
+         user_err Pp.(str "Program mode specified twice")
+       | VernacPolymorphic b when polymorphism = None ->
+         (Some b, atts)
+       | VernacPolymorphic _ ->
+         user_err Pp.(str "Polymorphism specified twice")
+       | VernacLocal b when Option.is_empty atts.locality ->
+         (polymorphism, { atts with locality = Some b })
+       | VernacLocal _ ->
+         user_err Pp.(str "Locality specified twice")
+       | VernacCoercion when not atts.coercion ->
+         (polymorphism, { atts with coercion = true })
+       | VernacCoercion ->
+         user_err Pp.(str "Coercion mode specified twice")
```

Comment:
I wonder how to raise these error messages. I haven't been able to trigger them. Could you add output test cases showing them?